src/main/kotlin/org/jitsi/nlj/RtpReceiverImpl.kt

```diff
@@ -107,6 +109,10 @@ class RtpReceiverImpl @JvmOverloads constructor(
     private val rtcpTermination = RtcpTermination(rtcpEventNotifier, logger)
     private val rembHandler = RembHandler(logger)
     private val toggleablePcapWriter = ToggleablePcapWriter(logger, "$id-rx")
+    private val videoBitrateCalculator = VideoBitrateCalculator(parentLogger)
+    private val audioBitrateCalculator = BitrateCalculator("Audio bitrate calculator")
+    override fun isReceivingAudio() = audioBitrateCalculator.bitrate >= 10.kbps
```

Comment:
Was there a calculation behind choosing 10 kbps as the "receiving" threshold, or is this just a number that felt  right?

Reply:
Kind of, but nothing formal. With a 5 second window and assuming a 50 kbps audio stream, this will flip 4 seconds after muting (and then 1 second after unmuting), which seemed reasonable.